Parsers/ASimAuthentication/Parsers/ASimAuthentication.yaml

```diff
@@ -47,7 +47,8 @@ ParserQuery: |
     ASimAuthenticationGoogleWorkspace (ASimAuthenticationDisabled or ('ExcludeASimAuthenticationGoogleWorkspace' in (DisabledParsers) )),
```

Comment:
Please remove Authentication files from this PR

Reply:
Change done.